### PR TITLE
[11.0][FIX] hr_timesheet_sheet: "Refuse" button name

### PR DESCRIPTION
**Pull request by astirpe**
The "Refuse" button name is wrong, causing troubles when trying to change the button attributes.

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: hr_timesheet_sheet/views/hr_timesheet_sheet_views.xml [hunk 1]
```diff
@@ -27,7 +27,7 @@
                     <button name="action_timesheet_confirm" states="draft" string="Submit to Manager" type="object" class="oe_highlight"/>
                     <button name="action_timesheet_done" states="confirm" string="Approve" type="object" groups="hr_timesheet.group_hr_timesheet_user" class="oe_highlight"/>
                     <button name="action_timesheet_draft" states="done" string="Set to Draft" type="object" groups="hr_timesheet.group_hr_timesheet_user"/>
-                    <button name="action_timesheet_draft" states="confirm" string="Refuse" type="object" groups="hr_timesheet.group_hr_timesheet_user"/>
+                    <button name="action_timesheet_refuse" states="confirm" string="Refuse" type="object" groups="hr_timesheet.group_hr_timesheet_user"/>
```
Comment (tafaRU) [inline hr_timesheet_sheet/views/hr_timesheet_sheet_views.xml:30]: Where is defined the method `action_timesheet_refuse`? I grep a bit in the code but I cannot find it.

Comment (astirpe) [reply hr_timesheet_sheet/views/hr_timesheet_sheet_views.xml:30]: Oops :+1:

Comment (tafaRU) [reply hr_timesheet_sheet/views/hr_timesheet_sheet_views.xml:30]: :wink:

**Review by leemannd — APPROVED** [review]
Thanks

**Review by tafaRU — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]

**Pull request merged**